decidim-admin/config/locales/en.yml

```diff
@@ -7,7 +7,9 @@ en:
   decidim:
     admin:
       actions:
+        activate: Set as active
```

Comment:
Maybe we should use the same wording: Activate/deactivate instead of Set as active/deactivate